otlp/src/main/java/io/opentelemetry/example/otlp/OtlpExporterExample.java

```diff
@@ -39,8 +39,8 @@ public static void main(String[] args) throws InterruptedException {
         exampleSpan.setAttribute("good", "true");
         exampleSpan.setAttribute("exampleNumber", i);
         Thread.sleep(100);
-      } finally {
         histogram.record(System.currentTimeMillis() - startTime);
+      } finally {
         exampleSpan.end();
       }
```

Comment:
it would be nice to not lose the metric recording on exception, maybe:

```
      Span exampleSpan = tracer.spanBuilder("exampleSpan").startSpan();
      Context exampleContext = Context.current().with(exampleSpan);
      try (Scope scope = exampleContext.makeCurrent()) {
        counter.add(1);
        exampleSpan.setAttribute("good", "true");
        exampleSpan.setAttribute("exampleNumber", i);
        Thread.sleep(100);
      } finally {
        histogram.record(System.currentTimeMillis() - startTime, Attributes.empty(), exampleContext);
        exampleSpan.end();
      }
```